.ci/install_runtime.sh

```diff
@@ -162,3 +163,20 @@ if [ "$arch" == "aarch64" -a "${KATA_HYPERVISOR}" == "qemu" -a "${ENABLE_ARM64_U
 	#enable pflash
 	sudo sed -i 's|#pflashes|pflashes|' "${runtime_config_path}"
 fi
+
+if [ "$TEE_TYPE" == "tdx" ]; then
```

Comment:
Yes, infact we try to test TDX features at different level,  for the tests in `functional` directory, it may ensure TDX functon is ok at kata runtime level. For the tests in `integration` directory, we want test the feature in E2E stack which can support different combination with different config: 
https://github.com/kata-containers/ci/tree/main/jenkins/jobs

We want test TDX stack works well in confidential container E2E pipeline like image downloading in guest/ image decryption/signature verification and more tests to be added.

Reply:
Hi @arronwy, I think that now I understood the goal of this PR. I'm okay with having the installation and configuration at the job setup stage.